Dockerfile

```diff
@@ -1,11 +1,33 @@
-FROM python:2.7 
+# getting the base Image
+FROM python:3.6
+
 MAINTAINER Ana Balica <ana.balica@gmail.com>
+
+# Updated by
+LABEL HUSSAIN="husainattar110@gmail.com"
+
 EXPOSE 8000
 
+# Initializing the working directory
 WORKDIR /usr/src
+
 RUN mkdir portal
+
 RUN cd portal
+
+# Re-Initializing the working directory
 WORKDIR /usr/src/portal
+
+# Copy the requirement.txt for installing
 COPY requirements/prod.txt /usr/src/portal/requirements/prod.txt
-RUN pip install -r requirements/prod.txt
+COPY requirements/dev.txt /usr/src/portal/requirements/dev.txt
+
+# Installing the requirements
+RUN pip install -r requirements/dev.txt
 
+# Installing the GDAL Library
+# RUN apt-get install python-gdal
+RUN apt-get update 
+RUN apt-get install -y software-properties-common
+RUN add-apt-repository ppa:ubuntugis/ppa 
+RUN apt-get install -y gdal-bin python-gdal python3-gdal
```

Comment:
Do we need `python-gdal`?

Reply:
Yes because while building it gives me an error regarding gdal lib not found during executing the cities_light command. So I tried to include and it resolved error.

Reply:
Got you!

Reply:
> Yes because while building it gives me an error regarding gdal lib not found during executing the cities_light command. So I tried to include and it resolved error.

are you sure about "python-gdal" as image using here is python3.6 , then why we need "python-gdal" library of python 2 .
As,  gdal lib not found during executing the cities_light command can be fixed by only installing "python3-gdal".

Also , installing "python-gdal" here will also install python2 and other dependency in the image .

So, My suggestion is to only use "python3-gdal" here and verify for issue and also sure image size will further reduce.